tests/unit/utils.spec.js

```diff
@@ -95,15 +95,6 @@
           assert.deepEqual(data, response.success);
         });
 
```

Comment:
This test cases used to cover when a user passed a wrong glob when specifying rules.
```yaml
- group: user
  permissions:
    - resource: "%"
      methods:"$"
      action: allow
```
Previously this was handled inside the middleware. I moved this functionality to `checkPropeties` method  and added test cases for each. Refer to line 259-318 in utils.pec.js.